extensions/circle-ci/src/index.tsx

```diff
@@ -0,0 +1,309 @@
+    showToast(ToastStyle.Failure, `Could not load pipelines due to ${error}`);
+    return Promise.resolve([]);
+  }
+}
+
+function getWorkflowAccessoryTitle(workflow: Workflow): string {
+  const createdAt = new Date(workflow.created_at).toLocaleString()
+  const stoppedAt = new Date(workflow.stopped_at).toLocaleString()
+  switch (workflow.status) {
+    case WorkflowStatus.success:
+      return `Succeeded at ${stoppedAt}`;
+    case WorkflowStatus.running:
+      return `Running since ${createdAt}`;
+    case WorkflowStatus.not_run:
+      return "Not run";
+    case WorkflowStatus.failed:
+      return `Failed at ${stoppedAt}`;
+    case WorkflowStatus.error:
+      return `Error at ${stoppedAt}`;
+    case WorkflowStatus.failing:
+      return `Failing since ${createdAt}`;
+    case WorkflowStatus.on_hold:
+      return `On hold since ${createdAt}`;
+    case WorkflowStatus.canceled:
+      return `Canceled at ${stoppedAt}`;
+    case WorkflowStatus.unauthorized:
+      return `Unauthorized at ${stoppedAt}`;
+    default:
+      return "Unknown";
+  }
+}
+
+function getWorkflowAccessoryIcon(workflow: Workflow): ImageLike {
+  switch (workflow.status) {
+    case WorkflowStatus.success:
+      return { source: Icon.Checkmark, tintColor: Color.Green };
+    case WorkflowStatus.running:
+      return { source: Icon.Gear, tintColor: Color.Blue };
+    case WorkflowStatus.not_run:
+      return { source: Icon.Circle, tintColor: Color.SecondaryText };
+    case WorkflowStatus.failed:
+      return { source: Icon.XmarkCircle, tintColor: Color.Red };
+    case WorkflowStatus.error:
+      return { source: Icon.XmarkCircle, tintColor: Color.Orange };
+    case WorkflowStatus.failing:
+      return { source: Icon.XmarkCircle, tintColor: Color.Red };
+    case WorkflowStatus.on_hold:
+      return { source: Icon.Clock, tintColor: Color.Blue };
+    case WorkflowStatus.canceled:
+      return { source: Icon.XmarkCircle, tintColor: Color.SecondaryText };
+    case WorkflowStatus.unauthorized:
+      return { source: Icon.ExclamationMark, tintColor: Color.Red };
+    default:
+      return { source: Icon.Gear, tintColor: Color.Blue };
+  }
+}
+
+
+function getWorkflowActions(workflow: Workflow) {
+  const workflowUrl = `https://app.circleci.com/pipelines/workflows/${workflow.id}`;
+  if (!workflow.repository.target_repository_url) {
+    return (
+      <ActionPanel>
+        <PushAction title="Workflow's jobs list" target={<JobList workflow={workflow} />} />
+        <OpenInBrowserAction title="Open workflow" url={workflowUrl} />
+        <CopyToClipboardAction title="Copy workflow URL" content={workflowUrl} />
+      </ActionPanel>
+    )
+  } else {
+    return (
+      <ActionPanel>
+        <PushAction title="Workflow's jobs list" target={<JobList workflow={workflow} />} />
+        <OpenInBrowserAction title="Open workflow" url={workflowUrl} />
+        <CopyToClipboardAction title="Copy workflow URL" content={workflowUrl} />
```

Comment:
```suggestion
        <PushAction title="Workflow Job List" target={<JobList workflow={workflow} />} />
        <OpenInBrowserAction title="Open Workflow" url={workflowUrl} />
        <CopyToClipboardAction title="Copy Workflow URL" content={workflowUrl} />
```
Also icon is missing in the PushAction